deployment/ansible/roles/nyc-trees.app/defaults/main.yml

```diff
@@ -2,6 +2,9 @@
 app_home: /opt/app
 app_config_home: /etc/nyc-trees.d/env
 app_deploy_branch: "master"
+
+tiler_host: localhost
```

Comment:
I think that this variable should be pulled out into `groups_vars` like in https://github.com/azavea/nyc-trees/pull/270. For the Packer `group_vars` file, we can set it to `tiler.service.nyc-trees.internal`.